src/CaptureEventProducer/BUILD

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+load("//:orbit.bzl", "orbit_cc_library", "orbit_cc_test")
+
+package(default_visibility = ["//:__subpackages__"])
+
+licenses(["notice"])
+
+orbit_cc_library(
+    name = "CaptureEventProducer",
+    deps = [
+        "//src/GrpcProtos:capture_cc_proto",
+        "//src/GrpcProtos:producer_side_services_cc_grpc_proto",
+        "//src/GrpcProtos:producer_side_services_cc_proto",
+        "//src/OrbitBase",
+        "@com_github_cameron314_concurrentqueue//concurrentqueue",
```

Comment:
Mainly out of curiosity: Where can I lookup how these are transformed into the corresponding third_party dependency? Is this done by copybara?

Reply:
Excellent question. The copybara change is here: cl/412238973

Reply:
And for explanation:

Bazel/Blaze have a tool called buildozer which allows to make changes to BUILD files programmatically. Copybara has an integration for buildozer and that's what I use to make the adjustments. You can do things like "replace this one dependency in all target rules by this other dependency". "dependency" in that regard can even be a regular expression.

Reply:
Ah, I was surprised I didn't find it with code search, but it's not submitted yet. Thanks for clarifying!